src/main/java/reposense/authorship/FileResultAggregator.java

```diff
@@ -15,15 +16,17 @@
     /**
      * Returns the {@code AuthorshipSummary} generated from aggregating the {@code fileResults}.
      */
-    public static AuthorshipSummary aggregateFileResult(List<FileResult> fileResults, List<Author> authors) {
-        AuthorshipSummary authorContributionSummary = new AuthorshipSummary(fileResults, authors);
+    public static AuthorshipSummary aggregateFileResult(List<FileResult> fileResults, List<Author> authors,
+                                                            List<Format> formats) {
+        AuthorshipSummary authorContributionSummary = new AuthorshipSummary(fileResults, authors, formats);
         for (FileResult fileResult : fileResults) {
             for (LineInfo lineInfo : fileResult.getLines()) {
                 Author author = lineInfo.getAuthor();
                 if (!authors.contains(author)) {
                     continue;
                 }
                 authorContributionSummary.addAuthorContributionCount(author);
+                authorContributionSummary.addAuthorFileTypeContributionCount(author, fileResult.getFileType());
```

Comment:
cant this be together with `addAuthorContributionCount`?

Reply:
Ok, i've removed the separate ```addAuthorFileTypeContributionCount``` method and combined it with ```addAuthorContributionCount```